app/src/main/java/org/thoughtcrime/securesms/preferences/HelpSettingsActivity.kt

```diff
@@ -30,20 +31,31 @@ class HelpSettingsActivity: PassphraseRequiredActionBarActivity() {
 class HelpSettingsFragment: CorrectedPreferenceFragment() {
 
     companion object {
-        private const val EXPORT_LOGS = "export_logs"
-        private const val TRANSLATE = "translate_session"
-        private const val FEEDBACK = "feedback"
-        private const val FAQ = "faq"
-        private const val SUPPORT = "support"
-
-        private const val CROWDIN_URL = "https://getsession.org/translate"
+        private const val EXPORT_LOGS  = "export_logs"
+        private const val TRANSLATE    = "translate_session"
+        private const val FEEDBACK     = "feedback"
+        private const val FAQ          = "faq"
+        private const val SUPPORT      = "support"
+        private const val CROWDIN_URL  = "https://getsession.org/translate"
         private const val FEEDBACK_URL = "https://getsession.org/survey"
-        private const val FAQ_URL = "https://getsession.org/faq"
-        private const val SUPPORT_URL = "https://sessionapp.zendesk.com/hc/en-us"
+        private const val FAQ_URL      = "https://getsession.org/faq"
+        private const val SUPPORT_URL  = "https://sessionapp.zendesk.com/hc/en-us"
     }
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         addPreferencesFromResource(R.xml.preferences_help)
+
+        // String sub the summary text of the `export_logs` element in preferences_help.xml
+        var exportPref = preferenceScreen.findPreference<Preference>(EXPORT_LOGS)
```

Comment:
Those two properties can be changed to vals

Reply:
Done